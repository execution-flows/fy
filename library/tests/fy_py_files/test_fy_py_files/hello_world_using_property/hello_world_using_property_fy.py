"""fy
flow HelloWorld -> None:
    property greeting using constant
"""
# fy:start ===>>>
        # fy:end <<<===
        print(self._greeting)
