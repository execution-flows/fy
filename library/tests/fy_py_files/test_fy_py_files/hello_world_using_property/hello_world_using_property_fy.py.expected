"""fy
flow HelloWorld -> None:
    property greeting using constant
"""

from base.execution_flow_base import ExecutionFlowBase

from fy_py_files.test_fy_py_files.hello_world_using_property.property.using_constant_fy import (
    Greeting_UsingConstant_PropertyMixin,
)


# fy:start <<<===
class Helloworld_Flow(
    # Property Mixins
    Greeting_UsingConstant_PropertyMixin,
    # Base
    ExecutionFlowBase[None]
):
    def __call__(self) -> None:
        # fy:end <<<===
        print(self._greeting)
